app/src/main/res/values/strings.xml

```diff
@@ -9,6 +9,11 @@
     <string name="scanning_result">[Unnamed]</string>
     <string name="scanning_saved">Previous data</string>
     <string name="unknown">Unknown</string>
+    <string name="brew_list">Brews</string>
+    <string name="brew_card_value_gravity">%.3f SG</string>
+    <string name="brew_card_value_abv">"%.2f\%%"</string>
+    <string name="brew_card_value_duration">%s</string>
+    <string name="brew_start_to_end">From: %s - To: %s</string>
```

Comment:
perchè non riusi le stringhe già presenti, onde evitare duplicazione?

Reply:
Per tenere separato in caso dobbiamo fare delle modifiche in una zona precisa. 
E se pensi che si possano riutilizzare allora preferirei dare un nome generico visto che alcune hanno riferimenti al grafico o alla pillola ma sono solo unità di misura.